src/puppetlabs/trapperkeeper/services/authorization/authorization_core.clj

```diff
@@ -36,7 +40,10 @@
 (defn- method
   "Returns the method key of a given config map, or :any if none"
   [config-map]
-  (keyword (get-in config-map [:match-request :method] :any)))
+  (let [method-from-config (get-in config-map [:match-request :method] :any)]
```

Comment:
@dankreek, only other question I had on this PR is whether we should lower-case the method around here somewhere since we allow mixed case to pass initial validation and I'm not sure we normalize to lower-case anywhere else before we use the rule.